release_build_files/readme.md

```diff
@@ -571,6 +571,31 @@ code.
 
 ## Release Notes
 
+### 7.3.0
```

Comment:
Please put this into 7.2.0, we're putting together that version shortly today.

Reply:
Done (in #366).